### PR TITLE
[13.0][MIG] product_code_mandatory

### PR DESCRIPTION
**Pull request by jarroyomorales**
Standard migration to v13 of `product_code_mandatory`.

@etobella @AdriaGForgeFlow @LoisRForgeFlow

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
I think the "black, isort, prettier" changes are in the migration commit instead of the corresponding commit. 🤔

**Comment by jarroyomorales** [comment]
@MiquelRForgeFlow It is good now! Thank you!

**Review by MiquelRForgeFlow — APPROVED** [review]

File: product_code_mandatory/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Product Internal Reference as Required",
```
Comment (MiquelRForgeFlow) [inline product_code_mandatory/__manifest__.py]:
```suggestion
    "name": "Product Code Mandatory",
```
maybe

**Review by etobella — APPROVED** [review]
LGTM :+1:  No major changes

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**